workflow/5_adaptation/benefit_cost_ratio_estimations.py

```diff
@@ -0,0 +1,626 @@
+    option_df,
+    hazard_thresholds,
+    cost_multiplication_factors,
+    hazard_thresholds_column_name,
+    protection_type_name,
+):
+    for idx, (ft, cmf) in enumerate(
+        list(zip(hazard_thresholds, cost_multiplication_factors))
+    ):
+        folder_name = f"{protection_type_name}_{str(ft).replace('.','p')}"
+        risk_file = os.path.join(
+            results_path,
+            folder_name,
+            risk_filepath,
+        )
+        if os.path.isfile(risk_file) is True:
+            adapt_risk = pd.read_csv(risk_file)
+            adapt_ead_eael_df, adapt_ead_eael_columns = get_ead_eael(
+                adapt_risk,
+                asset_id,
+                hazard["hazard"],
+                hazard["hazard_type"],
+                rcps,
+                risk_type,
+                val_type,
+            )
+            # print (no_adapt_ead_eael_df.columns.values.tolist())
+            ead_eael_df, ead_eael_benefit_columns = get_ead_eael_benefits(
+                asset_id,
+                no_adapt_ead_eael_df.copy(),
+                adapt_ead_eael_df,
+                ead_eael_columns,
+                adapt_ead_eael_columns,
+            )
+
+            option_cost_df = option_df.copy()
+            option_cost_df[hazard_thresholds_column_name] = ft
+            option_cost_df["adapt_cost_npv"] = cmf * option_cost_df["adapt_cost_npv"]
+            option_cost_df["adapt_cost_unit"] = "J$"
+            option_cost_df["ead_cost_unit"] = "J$"
+            option_cost_df["eael_cost_unit"] = "J$/day"
+            ead_eael_df = pd.merge(
+                option_cost_df[
+                    [
+                        asset_id,
+                        "adaptation_option",
+                        hazard_thresholds_column_name,
+                        "adapt_cost_unit",
+                        "ead_cost_unit",
+                        "eael_cost_unit",
+                        "adapt_cost_npv",
+                    ]
+                ],
+                ead_eael_df,
+                how="left",
+                on=[asset_id],
+            ).fillna(0)
+            ead_eael_df = ead_eael_df[ead_eael_df["adapt_cost_npv"] > 0]
+            ead_eael_df = ead_eael_df[
+                [
+                    asset_id,
+                    "adaptation_option",
+                    hazard_thresholds_column_name,
+                    "adapt_cost_unit",
+                    "ead_cost_unit",
+                    "eael_cost_unit",
+                    "adapt_cost_npv",
+                ]
+                + ead_eael_benefit_columns
+            ]
+
+            ead_eael_results.append(ead_eael_df)
+
+    return ead_eael_results
+
+
+@click.command()
+@click.version_option("1.0")
+@click.option(
+    "--asset-data",
+    "-a",
+    required=True,
+    type=click.Path(exists=True, dir_okay=False, file_okay=True, readable=True),
+    help="Path to the asset layer data",
+)
+@click.option(
+    "--cost-file",
+    "-a",
```

Comment:
This is the same short label as for `--asset-data`

Reply:
Thanks! Should be fixed now.